controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -1433,6 +1502,20 @@ func getApiServerCRPlaceholder() (types.NamespacedName, error) {
 	return fakeHco, nil
 }
 
+func getConfigMapCRPlaceholder() (types.NamespacedName, error) {
+	fakeHco := types.NamespacedName{
+		Name: configMapCRPrefix + randomConstSuffix,
+	}
+	namespace, err := hcoutil.GetOperatorNamespaceFromEnv()
+
+	if err != nil {
+		return fakeHco, err
+	}
+	fakeHco.Namespace = namespace
+
+	return fakeHco, nil
+}
+
```

Comment:
I think we should have a special handling for the rate limit CM, What do you think about returning the new namespaced value only for this CM, and for the rest of the CMs, return the one that used for "other"? you can pass the already computed value as a parameter to this function.

Reply:
I have addressed this using a different approach. I didn't find convenient modifying the function since `isTriggerdByConfigMap()` and `isTriggeredByHyperconverged` are using this function too. Please, let me know if the approach used is acceptable and clean. Thanks.

Reply:
You actually did the right logic in the `add` function :+1: 

I think that what confused me was the name of this function, and the related triggerdBy constant. Just "configMap" or "CM" is maybe too general. Could you rename them?

e.g. `getRateLimitConfigMapPlaceholder` or something like that. 
and also `triggeredByRateLimitCM`, for example. I'll add a comment there too.

Reply:
What do you think about `getStaticRateLimitConfigMapPlaceholder()` and `triggeredByStaticRateLimitCM`? Naming is sometimes hard :smiling_face_with_tear: